src/_DataStructures_/Heaps/MaxHeap/index.js

```diff
@@ -0,0 +1,78 @@
+class MaxHeap {
+  constructor() {
+    this.heap = [];
+  }
+
+  add(element) {
+    this.heap.push(element);
+    // check for the parent element & swap if required
+    // eslint-disable-next-line no-underscore-dangle
+    this.__traverseUpAndSwap(this.heap.length - 1);
+  }
+
+  getMax() {
+    return this.heap[0] || null;
+  }
+
+  remove() {
+    // return the element at the root
+    const max = this.heap[0] || null;
+    if (this.heap.length > 1) {
+      // move the leaf to the root
+      this.heap[0] = this.heap[this.heap.length - 1];
+      this.heap.splice(this.heap.length - 1, 1);
+      // restore the heapify property
+      // eslint-disable-next-line no-underscore-dangle
+      this.__heapify(0);
+      return max;
+    }
+
+    if (this.heap.length === 1) {
+      this.heap.splice(this.heap.length - 1, 1);
+      return max;
+    }
+
+    return max;
```

Comment:
Can you remove this line because it will never execute and it will increase code coverage.

Reply:
ok